From 64ff72faa9ec2460303bc48ee28e79771d2f2986 Mon Sep 17 00:00:00 2001
From: stvn <stvn@mit.edu>
Date: Tue, 8 Dec 2020 09:13:20 -0800
Subject: [PATCH 0325/1021] Run coverage via Github Action

Note: I tried directly invoking `codecov`, but it "couldn't tell" that
it was running in a GA/CI container, so I've opted for using the GA
codecov integration, which does work as expected.
---
 .github/workflows/validate.yml | 4 ++++
 Makefile                       | 1 -
 2 files changed, 4 insertions(+), 1 deletion(-)

diff --git a/.github/workflows/validate.yml b/.github/workflows/validate.yml
index a647b07..4bfaf73 100644
--- a/.github/workflows/validate.yml
+++ b/.github/workflows/validate.yml
@@ -15,3 +15,7 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
     - run: make validate.ci
+    - name: Upload coverage
+      uses: codecov/codecov-action@v1
+      with:
+        fail_ci_if_error: true
diff --git a/Makefile b/Makefile
index 9503197..e8b422a 100644
--- a/Makefile
+++ b/Makefile
@@ -66,4 +66,3 @@ validate:
 validate.ci:
 	npm ci
 	make validate
-	codecov
-- 
2.34.1

